openrtb_ext/imp_insticator.go

```diff
@@ -0,0 +1,7 @@
+package openrtb_ext
+
+// ExtImpInsticator defines the contract for bidrequest.imp[i].ext.prebid.bidder.insticator
+type ExtImpInsticator struct {
+	AdUnitId    string `json:"adUnitId,omitempty"`
+	PublisherId string `json:"publisherId,omitempty"`
```

Comment:
`static/bidder-params/insticator.json` mentions `AdUnitId PublisherId` as required param. Should not mark it as omitempty